.travis.yml

```diff
@@ -19,4 +19,4 @@ before_script:
   - composer install --prefer-source --no-interaction
```

Comment:
I think we need `--dev` flag if we want to use PHPUnit installed by Composer:
`composer install --prefer-source --no-interaction --dev`